docs/Running-Tachyon-with-RDMA.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: global
+title: Running Tachyon with RDMA over JXIO
+---
+
+## Prerequisite
+
+* **Compile JXIO**. Download and compile [JXIO](https://github.com/accelio/JXIO/).
+Tachyon is supported by JXIO v1.3 and above
+* **Install JXIO**.
+
+    mvn install:install-file -Dfile=jxio.jar -DgroupId=org.accelio -DartifactId=jxio -Dversion=<version> -Dpackaging=jar
```

Comment:
yes, found in bin/ folder afetr compilation
i'll update the doc to make it more clear